connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/outbound/ConnectorJobHandler.java

```diff
@@ -195,30 +220,16 @@ public void handle(final JobClient client, final ActivatedJob job) {
               new ErrorExpressionJobContext(new ErrorExpressionJob(job.getRetries())))
           .ifPresentOrElse(
               error -> {
-                if (error instanceof BpmnError bpmnError) {
-                  LOGGER.debug(
-                      "Throwing BPMN error for job {} with code {}",
-                      job.getKey(),
-                      bpmnError.code());
-                  throwBpmnError(client, job, bpmnError);
-                } else if (error instanceof JobError jobError) {
-                  LOGGER.debug("Throwing incident for job {}", job.getKey());
-                  failJob(
-                      client,
-                      job,
-                      new ErrorResult(
-                          Map.of("error", jobError.message()),
-                          new RuntimeException(jobError.message()),
-                          jobError.retries(),
-                          jobError.retryBackoff()));
-                }
+                handleFEELError(client, job, error);
```

Comment:
I extracted with a (hopefully) meaningful name so that a reader understands immediately the different kinds of errors/exceptions that can happen (FEEL vs SDK)

Reply:
TBH I'd rather keep calling it `throwBpmnError` as this refers to the concept of Error in a BPMN process